### ISSUE TITLE
Program is not hidden in system tray after windows startup

### ISSUE THREAD

**Issue by Acuario4** [rocksdanister/lively#1366]
## How to recreate:
- Set 'Start with windows'* in general settings to true.
- Set 'System tray symbol'* in general settings to true.
- Shutdown the computer and start it again (don't restart it).

<sub>_*Some settings' names may not be perfectly accurate as I use the program in a different language_</sub>
## Expected:
The program starts and displays my wallpaper as intended. It is nowhere to be seen in Windows' task bar.

## Reality:
The program starts and displays my wallpaper as intended. Lively Wallpaper's icon is displayed in Windows' task bar.

**Comment by rocksdanister** [comment]
>Set 'System tray symbol'* in general settings to true.
<img width="539" alt="image" src="https://user-images.githubusercontent.com/17554161/187208152-dc042d81-fda0-4992-82b4-4945dbb1f8f8.png">

This setting? it has to be OFF to hide taskbar icon.

**Comment by Acuario4** [comment]
> This setting? it has to be OFF to hide taskbar icon.

If I turn this setting on, the icon is shown in the system tray. If I turn the setting off, the icon is not shown in the system tray.
But either way, the icon is (also) displayed in the task bar (which is the problem).
I would like the icon to be visible in the system tray, but not in the task bar.

**Comment by Acuario4** [comment]
> The program starts and displays my wallpaper as intended. It is nowhere to be seen in Windows' task bar.

In addition, I just noticed that it works as expected whenever the computer is restarted.
If it is properly shutdown though, the problem occurs upon startup as described.

**Comment by rocksdanister** [comment]
>But either way, the icon is (also) displayed in the task bar (which is the problem).

Show a screenshot of the problem

**Comment by Acuario4** [comment]
> Show a screenshot of the problem

![image](https://user-images.githubusercontent.com/47208805/187395755-ddd0c099-9696-450c-a608-9ed5e29aed5d.png)

**Comment by rocksdanister** [comment]
I am not sure which Lively component that is showing the icon (UI or Web player.).. there also appears to be a loading indicator 🤔

Next time the problem happens share the logfile:
https://github.com/rocksdanister/lively/wiki/Common-Problems#having-a-problem-how-do-i-get-help--how-to-retrieve-log-files

**Comment by Acuario4** [comment]
> I am not sure which Lively component that is showing the icon (UI or Web player.).. there also appears to be a loading indicator 🤔

The loading indicator shows the progress of the video that I use as a wallpaper (I think).

> Next time the problem happens share the logfile: https://github.com/rocksdanister/lively/wiki/Common-Problems#having-a-problem-how-do-i-get-help--how-to-retrieve-log-files

Sure, I'll do so later

**Comment by Acuario4** [comment]
> Next time the problem happens share the logfile:

[lively_log_20220901_085120.zip](https://github.com/rocksdanister/lively/files/9467515/lively_log_20220901_085120.zip)
Here you go

**Comment by Acuario4** [comment]
Hey, is there anything else you need?
If there is, just let me know :)

**Comment by rocksdanister** [comment]
Its the mpv player, try these https://github.com/rocksdanister/lively/issues/598#issuecomment-877803702

**Comment by Acuario4** [comment]
Alright, I changed to the wmf player. I just started my PC and it seems to have fixed my problem.
Thank you very much for your really quick help :)
If I don't write anything here again the problem doesn't occur anymore.
However, I won't mark this issue as closed since this is only a workaround and not a real fix.
Just tell me when you think you have fixed the issue and I will gladly use the mpv player again to help testing.

**Comment by rocksdanister** [comment]
No, like the original issue its random.. wmf player is no longer included with Lively and switching it will just revert back to mpv.

**Comment by Acuario4** [comment]
Well, the problem hasn't occured anymore since I switched to wmf even though it is no longer included.

**Comment by ptanmay143** [comment]
Sorry for bumping an old issue but I currently am facing the same issue. The lively icon will persist in the Windows taskbar with a progress bar that is in-sync with the video wallpaper. Earlier when I faced this issue, I was switching to wmf, although now there is an error showing the plugin is not installed.

**Comment by TheQuickFox** [comment]
This issue still happens randomly with me. Windows 11 22H2, Lively v. 2.0.4.0, MPV player.

**Comment by cl37167** [comment]
I am having the same problem in Windows 10, including the wmf plugin not being installed.
